utils.py

```diff
@@ -0,0 +1,487 @@
+import csv
+import glob
+import multiprocessing as mp
+import os
+from typing import Tuple
+import warnings
+
+from absl import logging
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import scipy.io
+import tqdm    
+import xlmhg
+
+import istarmap
+
+
+warnings.simplefilter(action='ignore', category=FutureWarning)
```

Comment:
should be part of main? not sure anyone that imports this module would be interested in this filter